DiscordBot/Services/AudioService.cs

```diff
@@ -29,7 +29,7 @@ private async Task ConnectAudioClient()
             _audioClient = await channel.ConnectAsync();
         }
 
-        public async void Music()
+        public async Task Music()
```

Comment:
Do we still need the AudioService?  Could probably be stripped from the project.

Reply:
We can remove it. Let's remove everything useless.